Intersect.Client/Localization/Strings.cs

```diff
@@ -1787,6 +1787,15 @@ public partial struct Settings
 
             [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
             public static LocalizedString Vsync = @"V-Sync";
+            
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+            public static LocalizedString UiExpToPercentage = @"EXP to Percentage";
```

Comment:
`Show Experience as percentage`

Reply:
`Show Experience as percentage` it too long and hard to fit in the graphic image-box that belongs to the interface settings ; would you agree with `Experience as percentage` instead?